Test/Utils/WinContainers/Containers.ps1

```diff
@@ -58,22 +58,26 @@ function Remove-Container {
 
 
 function New-Container {
-    Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session,
+    Param ([Parameter(Mandatory = $true)] [Testbed] $Testbed,
            [Parameter(Mandatory = $true)] [string] $NetworkName,
            [Parameter(Mandatory = $false)] [string] $Name,
-           [Parameter(Mandatory = $false)] [string] $Image = "microsoft/nanoserver",
+           [Parameter(Mandatory = $false)] [AllowNull()] [string] $Image = $null,
            [Parameter(Mandatory = $false)] [string] $IP)
 
+    if(($null -eq $Image) -or ('' -eq $Image)) {
```

Comment:
is `'' -eq $Image` really needed?

Reply:
Yes, because if You don't pass parameter it is an empty string.